gutenberg/acquire/text.py

```diff
@@ -7,17 +7,23 @@
 import os
 from contextlib import closing
 
+try:
+    from functools import lru_cache
+except ImportError:
+    from functools32 import lru_cache
```

Comment:
Is this due to a Python 2/3 difference?

If so, commenting which is which will make dropping Python 2 easier in the near future.

Reply:
Yes, `functools.lru_cache` was only added in Python 3. the functools32 package is a backport for Python 2.7 I'll add a comment.

Reply:
Done in e49912f.